### ISSUE TITLE
Warnings with gloss-latex

### ISSUE THREAD

**Issue by jspitz** [reutenauer/polyglossia#284]
These should be fixed before the release:

```
Package polyglossia Warning: No hyphenation patterns were loaded for `latex'
(polyglossia)                I will use \language=\l@nohyphenation instead on i
nput line 13.

Package polyglossia Warning: \setlocalhyphenmin useless for unknown language la
tex on input line 13.

Module polyglossia Warning: Language latex not found in language.dat.lua on inp
ut line 13
```

**Comment by jspitz** [comment]
This is a problem with all gloss files where there is no matching hyphenation pattern to the language name (e.g., hebrew, syriac). The hyphennames key is apparantly read too late (after the warning is issued).

**Comment by reutenauer** [comment]
Unfortunately I don’t think I’ll have time to look into it before tomorrow or Friday.

**Comment by bastien-roucaries** [comment]
On Wed, Sep 4, 2019 at 8:33 AM Jürgen Spitzmüller <notifications@github.com>
wrote:

> This is a problem with all gloss files where there is no matching
> hyphenation pattern to the language name (e.g., hebrew, syriac). The
> hyphennames key is apparantly read too late (after the warning is issued).
>
Could you give me better diagnostic ?

You means that the option hyphennames is read too late but after what ?

We could do a two stage parsing of setup of gloss file, but I need to know
what i need to synchronize.

Thanks by advance

**Comment by jspitz** [comment]
>
> Could you give me better diagnostic ?
>

I am afraid no.

You means that the option hyphennames is read too late but after what ?
>

If I knew this I would have fixed it ;-)

>

**Comment by wehro** [comment]
Maybe you should delete these two lines defining an initial value for `hyphennames`:
```
    #1 / hyphennames
      .initial:x = {\c_empty_clist},
```

**Comment by jspitz** [comment]
> Maybe you should delete these two lines defining an initial value for `hyphennames`:
> 
> ```
>     #1 / hyphennames
>       .initial:x = {\c_empty_clist},
> ```

Yes, this fixes two of the warnings. With XeTeX, the `\setlocalhyphenmin` warning remains.

(Why) do we need the initial `hyphennames` value, @reutenauer ?

**Comment by jspitz** [comment]
Removing the initial hyphenmins value gets rid of the last warning as well (these lines:
```
    #1 / hyphenmins
     .initial:n = {2,3},
```
)

**Comment by bastien-roucaries** [comment]
Le dim. 8 sept. 2019 à 09:46, Jürgen Spitzmüller <notifications@github.com>
a écrit :

> Removing the initial hyphenmins value gets rid of the last warning as well
> (these lines:
>
>     #1 / hyphenmins
>      .initial:n = {2,3},
>
> )
>
> —
>


So we need a two stage setup. Removing is a work arround not a solution

**Comment by wehro** [comment]
With French, I now get the following warning several times:
```
Package polyglossia Warning: Asking to add empty feature to latin font(Script="
Latin" to scriptag "") on input line 5.

```

**Comment by jspitz** [comment]
> With French, I now get the following warning several times:
> 
> ```
> Package polyglossia Warning: Asking to add empty feature to latin font(Script="
> Latin" to scriptag "") on input line 5.
> ```

Fixed at 4151bd4
